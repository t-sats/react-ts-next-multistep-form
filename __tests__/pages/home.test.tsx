import React from 'react';
import { render } from '@testing-library/react';
import Home from '@/pages/index';

it('renders Home page correctly', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})
