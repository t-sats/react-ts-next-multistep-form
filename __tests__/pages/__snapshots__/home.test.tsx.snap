// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home page correctly 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-4"
  >
    <div
      class="flex gap-8"
    >
      <div
        class="flex flex-col gap-4 m-[10px]"
      >
        <p>
          Car Insurance
        </p>
        <a
          class="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#28a745] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
          href="/insurance/car_insurance"
          id="car_ins"
        >
          Get a quote
        </a>
      </div>
      <div
        class="flex flex-col gap-4 m-[10px]"
      >
        <p>
          Pet Insurance
        </p>
        <a
          class="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#28a745] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
          href="/insurance/pet_insurance"
          id="pet_ins"
        >
          Get a quote
        </a>
      </div>
    </div>
  </main>
</div>
`;
